packages/server/src/fhir/operations/projectclone.ts

```diff
@@ -67,7 +73,17 @@ class ProjectCloner {
   }
 
   rewriteIds(resource: Resource): Resource {
-    return JSON.parse(JSON.stringify(resource, (k, v) => this.rewriteKeyReplacer(k, v)));
+    const resourceObj = JSON.parse(JSON.stringify(resource, (k, v) => this.rewriteKeyReplacer(k, v)));
+
+    if (this.projectName) {
+      if (resource.resourceType === 'Project') {
+        resourceObj.name = this.projectName;
+      } else if (resource.resourceType === 'ProjectMembership') {
+        resourceObj.project.display = this.projectName;
+      }
```

Comment:
Customer requested that we rename the "default" `ClientApplication` as well.  It's a little gross, because it involves some string matching, but It would be valuable.

Here is where we create the default client: https://github.com/medplum/medplum/blob/main/packages/server/src/auth/newproject.ts#L81-L86

```ts
  const client = await createClient(systemRepo, {
    project,
    name: project.name + ' Default Client',
    description: 'Default client for ' + project.name,
  });
```

So the check would be something like:

```ts
} else if (resource.resourceType === 'ClientApplication' && resource.name.endsWith(' Default Client')) {
  // Update name
  // Update description
}
```

If that feels too loose, then we could skip for now, and add a proper boolean flag on the `ClientApplication` resource type that represents "this is the default client for the project".